Code/GraphMol/MMPA/MMPA.cpp

```diff
@@ -293,8 +294,81 @@ static void addResult(std::vector<std::pair<ROMOL_SPTR, ROMOL_SPTR> >&
       }
     }
   }
-  if (!resFound)
+  if (!resFound) {
+    //std::cerr << "**********************" << std::endl;
+    // From rfrag.py
+    // now change the labels on sidechains and core
+    // to get the new labels, cansmi the dot-disconnected side chains
+    // the first fragment in the side chains has attachment label 1, 2nd: 2, 3rd: 3
+    // then change the labels accordingly in the core
+    std::map<unsigned int, int> canonicalAtomMaps;
+    if( side_chains.get() ) {
+      RWMol _side_chain(*(side_chains.get())); // canonicalize the context or the core?
```

Comment:
starting identifiers with an underscore is technically a no-no. It's also not necessary here.